stable/mail2most/values.yaml

```diff
@@ -0,0 +1,18 @@
+deployment:
+  replicas: 1
+  image:
+    repository: virtomize/mail2most
+    tag: latest
```

Comment:
please dont use latest, does this image have a release or a tag version? if not consider to make it